examples/03_blade/analysis_options_aero.yaml

```diff
@@ -0,0 +1,235 @@
+general:
```

Comment:
It looks like these three files come from the WISDEM `examples` folder and the `03` designation would overlap with the existing WEIS `03` example.